### PR TITLE
Global page: Add Global page charts

### PR DESCRIPTION
**Pull request by ronyyosef**
Add the two charts to the global page
Filter options will be added in different PR
ChartJS: https://www.chartjs.org/docs/latest/
image

### CODE REVIEW

File: scripts/create_mock_data.py [hunk 1]
```diff
@@ -4,8 +4,9 @@
 
 
 def run():
-    for _ in range(500):
+    for i in range(500):
         house = HouseFactory()
         for _ in range(5):
             ExpenseFactory(house=house, month=random.randint(1, 12)).save()
+        print(i)
```
Comment (ShellyGolden) [inline scripts/create_mock_data.py:11]: this line is necessary?

File: static/css/global.css [hunk 2]
```diff
@@ -63,3 +63,18 @@
...
+  position: relative;
+  margin: 5px;
+}
+
+.chart_box {
+  position: relative;
+  height: 50vh;
+  width: 40vw;
+  margin-right: 30px;
+}
```
Comment (ShellyGolden) [inline static/css/global.css:80]: you need to add new line at the end of the file

**Review by ShellyGolden — COMMENTED** [review]
looks good
I left comments
please change the title of the PR

File: scripts/create_mock_data.py (hunk 1 above)
Comment (ronyyosef) [reply scripts/create_mock_data.py:11]: Nope, my bad

**Comment by ronyyosef** [comment]
I will add tests for the query soon

File: templates/global/chart.html [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+                    labels: [{%for category in categories%}
+                        '{{category}}',
+                    {%endfor%}],
+                        datasets: [
+                        {
+                            label: "Avg Amount",
+                            data: [{%for amount in amounts%}
+                                '{{amount}}',
+                            {%endfor%}],
+                            backgroundColor: randomColors,
```
Comment (LiorKGOW) [inline templates/global/chart.html:28]: Every expense gets a different random color,
Very nice

**Review by LiorKGOW — CHANGES_REQUESTED** [review]

**Review by shlior7 — COMMENTED** [review]
I think we can also add the tests from the draft 
Or at least some of them

https://github.com/redhat-beyond/Xpense/blob/fd9f97d03e20e57581f4d8272a0805f9506c0e7f/tests/global_tests.py

**Comment by ronyyosef** [comment]
I added the tests

**Review by orIsrael — APPROVED** [review]

**Review by ShellyGolden — APPROVED** [review]

**Review by svatares — APPROVED** [review]

**Pull request merged**